Dockerfile.foreman

```diff
@@ -0,0 +1,18 @@
+FROM python:3.5-slim
+
+RUN groupadd user && useradd --create-home --home-dir /home/user -g user user
+WORKDIR /home/user
+
+ENV CELERY_VERSION 4.0.2
+
+COPY workers/requirements.in .
+
+RUN pip install pip-tools && \
+    pip-compile requirements.in && \
+    pip install -r requirements.txt
```

Comment:
I see that there is this ``requirements.txt`` file referenced here, but I do not see it added in the initial commit. Will it be added later? Also, I am assuming this file is what will hold the version numbers for the different dependencies (as opposed to the requirements.in file, which just holds the file names)?